compute/k8s-services/dependencies.tf

```diff
@@ -197,3 +197,20 @@ data "aws_iam_policy_document" "cloudwatch_metrics_trust" {
     actions = ["sts:AssumeRole"]
   }
 }
+
+# --------------------------------------------------
+# Traefik dashboard secure access
+# --------------------------------------------------
+
+locals {
```

Comment:
Can we add some comments on what this logic does? At least it wasn't obvious to my afternoon brain :)

Reply:
Yes, that is understandable :)

I added the explaination in the PR details:

Ingress hostname logic:

If the traefik_alb_auth_core_alias has an list item with the value traefik.dfds.com (like it will have in production once the eks-pipeline is updated), the ingress hostname will be traefik.dfds.com.

Else (for other environments), the ingress hostname will be internal.cluster-name.capability-name.dfds.cloud